resources/public/api/conf/1.0/examples/amend_foreign_property_response.json

```diff
@@ -0,0 +1,19 @@
+{
+  "links":[
+    {
+      "href":"/Individuals/business/property/{nino}/{businessId}/period/{submissionId}",
+      "method":"GET",
+      "rel":"retrieve-property-period-summary"
+    },
+    {
+      "href":"/Individuals/business/property/{nino}/{businessId}/period/{submissionId}",
+      "method":"PUT",
+      "rel":"amend-property-period-summary"
+    },
+    {
+      "href":"/Individuals/business/property/{nino}/{businessId}/period",
+      "method":"GET",
+      "rel":"self"
+    }
+  ]
+}
```

Comment:
I feel the url values for nino, businessId and submissionId should have actual values in them to show a proper response example.